frontend/src/Components/MainContainer/MainContainer.tsx

```diff
@@ -16,6 +17,9 @@ const MainContainer = () => {
   const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
   const [selectedQuestion, setSelectedQuestion] = useState<string>("");
   const [selectedAnswer, setSelectedAnswer] = useState<string>("");
+  const [inputValue, setInputValue] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
```

Comment:
errorMessage is unused. Either incorporate it into the modal or remove the state. If you think this falls outside of the scope of this story, this feature will be moved into the next. But in this case, use a browser alert to tell the user, whether their answer is true or not